oak_runtime/src/channel.rs

```diff
@@ -280,6 +288,7 @@ impl Channel {
             reader_count: AtomicU64::new(0),
             waiting_threads: Mutex::new(HashMap::new()),
             label: label.clone(),
+            name: name.to_owned(),
```

Comment:
For node names, we presently combine the passed name with the id. 

https://github.com/project-oak/oak/blob/ab7e7f032359e0233ef3f859a9193733a9895372/oak_runtime/src/lib.rs#L1207

Should we do the same for channels?

Reply:
I think we should too.

Reply:
Done.